ftest.sh

```diff
@@ -270,11 +271,14 @@ EOF
 mkdir -p ~/.config/avocado/sysinfo/
 cat <<EOF > ~/.config/avocado/sysinfo/commands
 ps axf
-cat /proc/mounts
 dmesg
 df -h
 EOF
 
+cat <<EOF > ~/.config/avocado/sysinfo/files
+cat /proc/mounts
```

Comment:
I've never used the _files_ `[sysinfo.collectibles]`, but I'd suspect it's just a list of filenames rather than commands to dump the file contents.
```suggestion
/proc/mounts
```
I suppose the test run will be the final say as to whether/how it works.